src/ploneorg/addonlisting/browser/templates/folder_view.pt

```diff
@@ -0,0 +1,168 @@
+<html
+   xmlns="http://www.w3.org/1999/xhtml"
+   xmlns:metal="http://xml.zope.org/namespaces/metal"
+   xmlns:tal="http://xml.zope.org/namespaces/tal"
+   xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+   i18n:domain="ploneorg.addonlisting"
+   metal:use-macro="context/main_template/macros/master">
+   <head>
+      <metal:block fill-slot="style_slot">
+         <link rel="stylesheet" type="text/css"
+            tal:define="navroot context/@@plone_portal_state/navigation_root_url"
+            tal:attributes="href string:${navroot}/++resource++ploneorg.addonlisting/addonlist.css"
+            />
+      </metal:block>
+   </head>
+   <body>
+      <metal:content-core fill-slot="content-core">
+         <dl class="dropdown">
```

Comment:
This might should be an included form that is autogenerated from the vocabularies.